saleor/account/events.py

```diff
@@ -75,39 +77,42 @@ def customer_downloaded_a_digital_link_event(
     )
 
 
-def staff_user_deleted_a_customer_event(
-    *, staff_user: UserType, deleted_count: int = 1
+def staff_or_app_deleted_a_customer_event(
```

Comment:
Maybe instead of adding `staff_or_app`  in the name, we could just name it `customer_deleted_event`? 
Also, I would consider adding single args which will accept User or App (like requestor), then before we will create new event  (like we did in line 85), we can determine if the requestor is an app or user. This should simplify the usage of this function - like I don't need to fetch user and app separatly. I will just fetch the requestor which called this mutation, and pass it to the event function. The event function will handle the rest.